### ISSUE TITLE
Configuration of status updates

### ISSUE THREAD

**Issue by mbillingr** [codecov/support#148]
Codecov seems to be an amazing tool and we want to use it in our projects. However, we have been running into various configuration issues. Unfortunately, our story is long and hard to reproduce. Please bear with me :-)

Codecov was remarkably simple to set up. After adjusting `.travis.yml` (scot-dev/scot@af44b32c713eed45bd15b78c6ddb439aa6a8ec80) we immediately got coverage reports and the Codecov Bot commented on our pull requests :+1:.
We did not get build-status updates on PRs until we added `.codecov.yml` (scot-dev/scot@1fa69aea5b0492c68f62e4e2cca366fb3a2d07f6) to the repository.

Everything seems to be working fine in this project. I then created a small project to play around with the codecov configuration: https://github.com/scot-dev/integration-tests. I made a PR into this project (scot-dev/integration-tests#1) - and no status updates, although there is a `.codecov.yml`. It seems presence of this file has nothing to do with the appearance of status after all.

Furthermore, the settings in `.codecov.yml` seem to be ignored. I've set comment behavior to `spammy` but the bot still updates the original comment.

I guess the story boils down to two questions:

1. How do you enable build status updates?
2. How do you get Codecov to use the configuration file?
3. (Or did I miss the point, entirely?)

Cheers,
Martin

**Comment by stevepeak** [comment]
Hey @mbillingr `.codecov.yml` is not supported yet. This is part of our next major release. So this file is essentially ignored till then.

Sorry for the confusion. We are working on this major release and hope to deploy in a month or so. 

Meanwhile you can adjust all setting for your repository in Codecov UI. Click the "gear" icon (top right corner of your repository page) to reveal all the settings that you may adjust.

Thank you!

**Comment by mbillingr** [comment]
Hi @stevepeak, thank you for the clarification. That explains part of my confusion :)

Any idea why we don't get commit status reports, although it is enabled in the UI settings?

**Comment by stevepeak** [comment]
Hey @mbillingr I'm seeing successful posts to scot-dev/scot commit status :) thanks for being awesome!

**Comment by mbillingr** [comment]
Sorry @stevepeak for not letting this rest.

I found the reason why I did not get status reports on some PRs in scot-dev/integration-tests.

Instead of pushing new commits I restarted the Travis build after making changes to the Codecov configuration (such as changing the limit for incremental coverage). These changes seem to be ignored when the build is restarted. Only after pushing a new commit the new configuration has an effect.

I don't know if this is known or intended behavior, so I thought I'd let you know.

Cheers
